src/components/Card.jsx

```diff
@@ -0,0 +1,136 @@
+import Arrow from "../assets/icon-arrow.svg";
+import { useEffect, useState } from "react";
+
+import FormInput from "./FormInput";
+import AgeResults from "./AgeResults";
+const monthsUtil = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
+const Card = () => {
+  const [day, setDay] = useState("");
+  const [month, setMonth] = useState("");
+  const [year, setYear] = useState("");
+  const [result, setResult] = useState({
+    year: "-",
+    month: "-",
+    days: "-",
+  });
+  const [error, setError] = useState({
+    error: false,
+    msj: "",
+  });
+
+  useEffect(() => {
+    if (day.length == 0 || month.length == 0 || year.length == 0) {
+      setError({
+        error: true,
+        msj: "This field is required",
+      });
+    } else if (
+      day <= 0 ||
+      day > 31 ||
```

Comment:
Can we validate the max number of day as a function of the month? ![image](https://user-images.githubusercontent.com/79494436/233380093-a3ad1d59-5ed0-4f90-871c-ef3d61eb1558.png)